Finalise governance docs - resolve outstanding queries (#23)
physiopy/physiopy-governance: org-docs/CHARTER.md

- Understand and agree with the code of conduct, the contributors guidelines, and the governance of the Organization, and promote them across the Organization
- Understand the goals and priorities of the Organization and promote them

> - Be actively involved in a project

Reviewer: Are both bullet points necessary, or is this are these either/or criteria?

Reviewer 2: I'd be keen in having this as either/or for the moment and make it a "both" once the community grows enough.

> - Be actively involved in a project

Reviewer 2: ```suggestion
- Be actively involved in a project, that is aiming to:
```

## 10. Contribution Recognition and Authorship

### Repository contributions

Physiopy uses the [all-contributors specification](https://allcontributors.org/overview/) to capture diverse contributions. For each repository on Github, a current list of contributors should be stored in the `.all-contributorsrc` file and displayed in a table in the root README. A type of contribution is reflected by its corresponding [emoji key](https://allcontributors.org/emoji-key/). The steering committee, and any project leads, will periodically check these contributor summaries for accuracy and also encourage each contributor to ensure their contributions are recognized appropriately (for each project repository and on website pages that summarize contributions across the organization).  

> ### Other pathways for recognition

Reviewer 2: ```suggestion
### Authorship on deliverables
```

> Physiopy will submit publications in various forms, such as posters, talks, pre-prints, journal submissions. In these cases, Physiopy will extend authorship to all those involved in the relevant development activity during the corresponding time frame. By default, authorship on these publications will be listed in alphabetical order. It is acceptable for anyone in this list of authors to suggest that certain authors be first or last due to their specific contributions; for this to be approved a consensus must be reached (everyone supports, or no individual strongly opposes). If a consensus cannot be reached, the steering committee will vote as per section 3.6. 

Reviewer 2: ```suggestion
Any deliverable that physiopy releases (e.g. software, online documentation, posters, scientific manuscript, ...) will have authorship reflecting all contributions to the deliverable itself, i.e. all those involved in the relevant development activity during the corresponding time frame. 

By default, authorship on these publications will be listed in alphabetical order.

It is acceptable to deviate from this order and suggest that certain authors be first or last due to their specific contributions; in this case a consensus must be reached (everyone supports, or no individual strongly opposes) as early in the project life as possible, and can be re-discussed at any point in versioned deliverables (e.g. software). If a consensus cannot be reached, the steering committee will vote as per section 3.6.
When possible, projects that do not default to alphabetical authorship should document authorship rules publicly. 

We generally suggest that the 2-3 people taking the lead on a project are listed first (as co-authors or in the order they prefer), and that particular contributions that reflects senior contributions (e.g. general organisation, financial arrangements, etc.) are listed last.

When possible, the list of authors should always end with a "consortia-like" authorship reflecting the community, that is "_The Physiopy Community_".

We recommend for all contributions to be reported, independently of current active status of involved contributors. However, consent of individual authors must be guaranteed at all times.
```

Reviewer: I appreciate the extra detail here, but I wonder why we have exceptions for software deliverables and for the community practices? Especially since the rules here are fairly flexible - I'm not sure we need to have any project-specific mentions in the charter.

Reviewer: I also think consensus cannot be guaranteed, especially since we discuss a mechanism for when it cannot be reached a few paragraphs earlier.

Author: We have discussed this in the steering committee meeting (3rd June 2026)
